test/development/app-dir/dynamic-error-trace/index.test.ts

```diff
@@ -25,18 +24,18 @@ createNextDescribe(
     startCommand: (global as any).isNextDev ? 'pnpm dev' : 'pnpm start',
     buildCommand: 'pnpm build',
     skipDeployment: true,
-  },
-  ({ next }) => {
-    it('should show the error trace', async () => {
-      const browser = await next.browser('/')
-      await hasRedbox(browser)
-      await expandCallStack(browser)
-      const callStack = await getRedboxCallStack(browser)
+  })
+  if (skipped) return
```

Comment:
Why is this early return only added sometimes?

Reply:
The previous pattern used by `createNextDescribe` selectively called the "callback" if the tests weren't to be skipped. Using the new `nextTestSetup`, it will return a boolean if the tests should be skipped or not. Ideally we can get rid of these entirely by just inspecting the `isNextDev` or similar to decide to skip.